requirements.txt

```diff
@@ -1,6 +1,17 @@
-pandas
-numpy
-seaborn
-matplotlib
-wordcloud
-python-dateutil
\ No newline at end of file
+beautifulsoup4==4.11.1
+contourpy==1.0.5
+cycler==0.11.0
+fonttools==4.37.4
+kiwisolver==1.4.4
+matplotlib==3.6.1
+numpy==1.23.4
+packaging==21.3
+pandas==1.5.0
+Pillow==9.2.0
+pyparsing==3.0.9
+python-dateutil==2.8.2
+pytz==2022.4
+seaborn==0.12.0
+six==1.16.0
+soupsieve==2.3.2.post1
+wordcloud==1.8.2.2
```

Comment:
There are some unused dependencies (e.g. seaborn).

Reply:
Ohh, I just used `pip3 freeze` to generate it.